ci-operator/config/openshift/cluster-api-actuator-pkg/openshift-cluster-api-actuator-pkg-master.yaml

```diff
@@ -100,6 +100,72 @@ tests:
         requests:
           cpu: 100m
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-aws-operator-periodic
+  cron: 0 2 * * WED,SAT
```

Comment:
Why not just every 72h? I think then the CI operator can schedule as it likes to avoid conflicts with other jobs

```
interval: 72h
```

Reply:
Done!